mods/default/nodes.lua

```diff
@@ -225,16 +225,6 @@ default:cloud
 
 --]]
 
--- Required wrapper to allow customization of default.after_place_leaves
```

Comment:
Use a clean workspace. These changes surely won't be accepted.

Reply:
Eh? How'd that get there! That's not my commit.

Reply:
I didn't modify the leaves or anything. I'll check my commit.

Reply:
My mistake; somehow I had merged a commit into it.